Library/OcAfterBootCompatLib/CustomSlide.c

```diff
@@ -374,6 +380,11 @@ ShouldUseCustomSlideOffset (
     }
   }
 
+  if (SlideSupport->ValidSlideCount < SlideSupport->ProvideMaxSlide) {
```

Comment:
I do not know what exactly this does, but it looks like some artifact of the previous change. Please remove.